### ISSUE TITLE
Flexible height

### ISSUE THREAD

**Issue by Madeirense** [mad-genius/mg-space#5]
Content overlaps container: 

![mg-space-height-issue-screenshot-1](https://cloud.githubusercontent.com/assets/4189203/22339388/b9e92be2-e3e1-11e6-938e-37601db1e2a3.jpg)
![mg-space-height-issue-screenshot-2](https://cloud.githubusercontent.com/assets/4189203/22339389/ba05d3aa-e3e1-11e6-99e7-222538a74e50.jpg)

**Comment by bmullican** [comment]
Paulo

Unfortunately the test page isn't working for me. I would be glad to help as soon as i can see the test page. I tried multiple browsers. I think it is a fullpage issue.

**Comment by Madeirense** [comment]
Hi Bryce,


Have a look at it here: https://dashnimorad.com/


Ignore the video issue. I have done it in another way. But click on the first tile on the page above. Sometimes it looks fine but other times the text overlaps the tiles bellow. If you don't see the issue immediately refresh the page and click on it again.


Thanks  a lot mate!

Dashni Morad<https://dashnimorad.com/>
dashnimorad.com
I am art. I am pain. I cry. I fight. I care. I am woman, and child too. I am darkness, yet still virtue.




________________________________
De: Bryce Mullican <notifications@github.com>
Enviado: sexta-feira, 27 de janeiro de 2017 15:28
Para: mad-genius/mg-space
Cc: madeirense; Author
Assunto: Re: [mad-genius/mg-space] Flexible height (#5)


Paulo

Unfortunately the test page isn't working for me. I would be glad to help as soon as i can see the test page. I tried multiple browsers. I think it is a fullpage issue.

-

**Comment by bmullican** [comment]
Can you replace the code from scripts.js from line 241 -> 273 with the following on the site above and lets test. This is a temporary fix like before. I will have to write something into the core plugin that deals with this dynamic height issue.

// Initialize Thumbnail expander
$('.mg-space-init').mgSpace({
  breakpointColumns: [
    {
      breakpoint: 0,
      column: 1
    },
    {
      breakpoint: 768,
      column: 3
    },
    {
      breakpoint: 992,
      column: 4
    }
  ]
});

// This will adjust for the height issues
$('.mg-space-init').on('afterOpenTarget', function (event, mgSpace, element) {
  setTimeout(function () {
    $('.mg-space').height($('.mg-target-open').height() + 120);
    console.log('Done'); 
  }, 300);
});

**Comment by Madeirense** [comment]
That did it mate! Please let me know if you wish to be paid for this.


Also just one more thing. When you click on a tile and the content opens bellow there is some automatic scrolling. How would I make it so that this scrolling always stops at the very top of the tiles instead of in the middle as it is now.


Tank you very much mate!


Paulo


________________________________
De: Bryce Mullican <notifications@github.com>
Enviado: sexta-feira, 27 de janeiro de 2017 17:10
Para: mad-genius/mg-space
Cc: madeirense; Author
Assunto: Re: [mad-genius/mg-space] Flexible height (#5)


Can you replace the code from scripts.js from line 241 -> 273 with the following on the site above and lets test. This is a temporary fix like before. I will have to write something into the core plugin that deals with this dynamic height issue.

// Initialize Thumbnail expander
$('.mg-space-init').mgSpace({
breakpointColumns: [
{
breakpoint: 0,
column: 1
},
{
breakpoint: 768,
column: 3
},
{
breakpoint: 992,
column: 4
}
]
});

// This will adjust for the height issues
$('.mg-space-init').on('afterOpenTarget', function (event, mgSpace, element) {
setTimeout(function () {
$('.mg-space').height($('.mg-target-open').height() + 120);
console.log('Done');
}, 300);
});

-

**Comment by bmullican** [comment]
Awesome. If you are referring to the fixed header that the thumbnails slide under when it scrolls to the top. I will write something for that.

Bryce

**Comment by Madeirense** [comment]
Yes that's the ones. It would be nice if the page would always scroll to the top of the thumbnails instead of stoping in the middle of the thumbnail.


Thanks a lot man. I really appreciate your help on this. Please let me know how much I owe you.


Paulo

________________________________
De: Bryce Mullican <notifications@github.com>
Enviado: sexta-feira, 27 de janeiro de 2017 18:23
Para: mad-genius/mg-space
Cc: madeirense; Author
Assunto: Re: [mad-genius/mg-space] Flexible height (#5)


Awesome. If you are referring to the fixed header that the thumbnails slide under when it scrolls to the top. I will write something for that.

Bryce

-
